app/controllers/rating_questions_controller.rb

```diff
@@ -1,27 +1,84 @@
 class RatingQuestionsController < ApplicationController
+  
+  before_action :find_question, only: [:show, :destroy, :edit, :update]
+  
   def index
     @rating_questions = RatingQuestion.all
+
+    # binding.pry
   end
 
   def new
     @rating_question = RatingQuestion.new
   end
 
   def create
-    @rating_question = RatingQuestion.new(question_params)
+    head 400 if !rating_question_params[:title]
+ 
+    @rating_question = RatingQuestion.new(rating_question_params)
+
     if @rating_question.save
-      # flash[:notice] = "Your question has been created."
-      redirect_to "/", notice: "Your question has been created."
+      respond_to do |format|
+        format.html { redirect_to '/', notice: "Your question has been created." }
+        format.json { render 'show', status: 201 }
+      end
+    else
+      errors = { 'errors' => @rating_question.errors.messages }
+
+      respond_to do |format|
+        format.html { redirect_to '/', alert: "Your question has NOT been created." }
```

Comment:
I think a better behaviour for the "fail" case here would be to re-render the `new` template and to display the flash message. We should present the user with a form containing whatever they've entered, and error messages indicating which fields have failed.